src/lib/helpers.js

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+
+const fs = require("fs")
+
+const addTrailingSlash = path => {
+  /*
+    Adds a trailing slash to a path if it doesn't have one
+    */
+  const lastChar = path.substr(path.length - 1)
+  if (lastChar !== "/") {
+    return `${path}/`
+  } else return path
+}
+
+const getCourseImageUrl = courseData => {
+  /*
+    Constructs the course image filename using parts of the course short_url
+    */
+  const courseNameParts = courseData["short_url"].split("-")
+  const imageName = `${courseNameParts[0]}-${
+    courseNameParts[1]
+  }${courseNameParts[courseNameParts.length - 2].charAt(0)}${courseNameParts[
+    courseNameParts.length - 1
+  ].slice(2)}.jpg`
+  courseData["course_files"].forEach(media => {
+    if (media["parent_uid"] === courseData["uid"]) {
+      const fileLocationParts = media["file_location"].split("/")
+      const jsonFile = fileLocationParts[fileLocationParts.length - 1]
+      const imageFile = `${media["uid"]}_${imageName}`
+      if (jsonFile === imageFile) {
+        return media["file_location"]
+      }
+    }
+  })
+  return "images/course_image.jpg"
```

Comment:
Is this true for each OCW file?

Reply:
It's not.  I put this in there as a fallback default because I had a brain fart about returning from forEach.  I altered the code so that this is just set on a variable as the default, then the forEach loop will set that variable if the file is actually found in the course_files collection.